src/backend/session/direct.rs

```diff
@@ -160,6 +162,25 @@ pub struct DirectSessionNotifier {
     source: Option<Signals>,
 }
 
+impl fmt::Debug for DirectSessionNotifier {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("Point")
+            .field("tty", &self.tty)
+            .field("active", &self.active)
+            .field("signaler", &self.signaler)
+            .field("signal", &self.signal)
+            .field("logger", &self.logger)
+            .field(
+                "source",
+                &match self.source {
+                    Some(_) => "Some(..)",
+                    None => "None",
+                },
+            )
+            .finish()
+    }
+}
+
```

Comment:
This one is missing a comment, I guess it's because `Signals` does not implement `Debug`?

Reply:
yep, I added a comment for that